src/screens/FeedScreen/RenderList.js

```diff
@@ -161,6 +165,7 @@ const RenderListFeed = (props) => {
             topics={item?.topics}
             item={item}
             onNewPollFetched={onNewPollFetched}
+            isHaveComment={isHaveComment}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `isHaveComment` is passed to the `Content` component. However, the prop name `isHaveComment` is grammatically incorrect and can be confusing. Consider renaming it to `hasComment` for better readability.
```diff
- isHaveComment={isHaveComment}
+ hasComment={hasComment}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->